### ISSUE TITLE
What is the advantage of continuous time gaussian?

### ISSUE THREAD

**Issue by Mut1nyJD** [lucidrains/denoising-diffusion-pytorch#32]
Unfortunately I can not spin really my head around it what the major benefit it has? Any short explanation?

**Comment by lucidrains** [comment]
![sample-48](https://user-images.githubusercontent.com/108653/172658920-1abfdb7e-0c31-4b5c-a338-dfe6b05cd94a.png)

certainly doesn't look any worse on my first experiment!

**Comment by Mut1nyJD** [comment]
> @Mut1nyJD it was suggested that it was used in the Imagen paper, if you look at their configs in the appendix
> 
> it also allows for a parameterized noise schedule (although it is still unclear at this point how beneficial that is)

Thank you I have to say I probably missed that bit in their paper I did not know it was proposed there first. So thank you for that short excerpt. 

I guess I have to try and see but at least your example show it does not make it worse but if it does not make it better or faster training process than hard to see a point in it :)  . 

I've added P2 weighting to the loss which has been proposed by some group as well still looking if that makes a change.

**Comment by lucidrains** [comment]
> > @Mut1nyJD it was suggested that it was used in the Imagen paper, if you look at their configs in the appendix
> > it also allows for a parameterized noise schedule (although it is still unclear at this point how beneficial that is)
> 
> Thank you I have to say I probably missed that bit in their paper I did not know it was proposed there first. So thank you for that short excerpt.
> 
> I guess I have to try and see but at least your example show it does not make it worse but if it does not make it better or faster training process than hard to see a point in it :) .
> 
> I've added P2 weighting to the loss which has been proposed by some group as well still looking if that makes a change.

interesting paper! let me know if you find any success with p2 weighting (and i'll add it to Imagen)
